src/prommis/solvent_extraction/sx_model_flowsheet_steady_state.py

```diff
@@ -7,20 +7,28 @@
 )
 from idaes.core.util.model_statistics import degrees_of_freedom as dof
 
+import matplotlib.pyplot as plt
+
+import numpy as np
+
 from prommis.leaching.leach_solution_properties import LeachSolutionParameters
 from prommis.solvent_extraction.ree_og_distribution import REESolExOgParameters
+from prommis.solvent_extraction.ree_aq_distribution import REESolExAqParameters
 from prommis.solvent_extraction.solvent_extraction import SolventExtraction
 
 m = ConcreteModel()
 m.fs = FlowsheetBlock(dynamic=False)
 m.fs.prop_o = REESolExOgParameters()
+m.fs.prop_a = REESolExAqParameters()
 m.fs.leach_soln = LeachSolutionParameters()
 
+number_of_stages=3
+
 m.fs.solex = SolventExtraction(
-    number_of_finite_elements=3,
+    number_of_finite_elements=number_of_stages,
     dynamic=False,
     aqueous_stream={
-        "property_package": m.fs.leach_soln,
```

Comment:
Does this flowsheet still work if you use the leaching prop pack? I believe we opted to use it before since it was identical/near-identical to the SX aqueous prop pack and wanted to use the same prop pack that leaching was using for its aqueous solution

Reply:
U have to deactivate the reaction terms for the dynamic scenario, like set the terms like has_reactions to False.. I think it should work for steady state

Reply:
This should not be an issue, and if it is it suggests and issue elsewhere in your model. You should be able to have these equilibrium reactions in parallel with everything else.

If you have trouble solving the model using the leaching properties, try using the `DiagnosticsToolbox` to see what the issue is.

Reply:
For steady state, the leaching property package can be used, I was talking about the dynamic scenario

Reply:
I was referring to dynamics as well. I cannot see any reason why you should not be able to solve the dynamic model with the reactions present.

Reply:
There was a DOF error if i remember correctly. I will recheck it